build/ci/evergreen.yml

```diff
@@ -1767,15 +1719,6 @@ buildvariants:
       go_bin: "/opt/golang/go1.16/bin"
     tasks:
       - name: release
-  - name: release_generate_docs
```

Comment:
since this no longer runs you need to update this file, https://github.com/mongodb/mongocli/blob/master/build/package/send-slack-notification.sh and remove the reference to the generated docs